### DISCUSSION TITLE
Get confidence of timestamps per word

### DISCUSSION THREAD

**Discussion by hashefa** [linto-ai/whisper-timestamped#84] [General]
Thank you for the great work.
If I understand correctly, confidence for each word is confidence for the tokens, am I correct?
I was wondering if getting confidence for each word's timing would be possible. 
Could be very helpful in my use case.

**Comment by hashefa** [comment]
@Jeronymous ?

**Comment by Jeronymous** [comment]
That's a good point  but I hardly see how to get a meaningful score for the timestamps, given that they are obtained in a "heuristic" way, and that timestamps probabilities given by Whisper model are not reliable at all from my experience.

But let's understand your need here.
You need a probability of the timestamps being right up to some precision (e.g. 100 ms)? Or a probability distribution of several timestamps 20 ms intervals? (20 ms being the maximum accuracy)?
How would you use such probabilities?
